extensions/integration-tests/src/main.ts

```diff
@@ -34,26 +35,71 @@ export function activate(context: vscode.ExtensionContext) {
 	});
 
 	vscode.commands.registerCommand('test.waitForExtensionsToLoad', async () => {
-		let expectedExtensions = ['Microsoft.agent', 'Microsoft.import', 'Microsoft.mssql', 'Microsoft.profiler'];
+		const expectedExtensions = ['Microsoft.agent', 'Microsoft.import', 'Microsoft.mssql', 'Microsoft.profiler', 'Microsoft.azuredatastudio-postgresql'];
+		const commonFeatures: azdata.DataProviderType[] = [
```

Comment:
How are we going to track updating this method? It would probably be beneficial to add another test before this that checks if any values were added to the enum that aren't captured here so we know to come in and update this